frontend/redux/actions.js

```diff
@@ -119,6 +119,12 @@ export const saveDemographicAnnotations = ({ id, demographics }) => ({
 });
 
 export const COMPLETE_WORKLOAD = 'COMPLETE_WORKLOAD';
-export const completeWorkload = () => ({
+export const completeWorkload = workload => ({
   type: COMPLETE_WORKLOAD,
+  payload: workload,
+});
+
+export const SEND_WORKLOAD = 'SEND_WORKLOAD';
+export const sendWorkload = () => ({
+  type: SEND_WORKLOAD,
```

Comment:
Doesn't look like this action creator is used, lines 127 onward should be able to be removed